packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -245,17 +245,24 @@ export interface ClusterProps {
   /**
    * Where to place the instances within the VPC
    *
-   * @default private subnets
+   * @default - private subnets
    */
   readonly vpcSubnets?: ec2.SubnetSelection;
 
   /**
    * Security group.
    *
-   * @default a new security group is created.
+   * @default - a new security group is created.
    */
   readonly securityGroups?: ec2.ISecurityGroup[];
 
+  /**
+   * The name of an existing Redshift cluster subnet group.
```

Comment:
are cluster subnet groups resources that can be imported? (i.e. they have an ARN, can live across accounts, etc) or are they just local to the account and region?

did you consider supporting this as an imported resource. i.e. `fromSubnetGroup` with an `IClusterSubnetGroup` that would contain the name within it?

Reply:
> are cluster subnet groups resources that can be imported? (i.e. they have an ARN, can live across accounts, etc) or are they just local to the account and region?

They don't have ARNs (https://docs.aws.amazon.com/redshift/latest/APIReference/API_ClusterSubnetGroup.html); as far as I can tell, they are local to account/region. We probably could create an L2 for them, although there's no backlog item (or visible demand) for that. I figured creating the L2 just for the `fromSubnetGroupName` import method was overkill in this case. We can always later deprecate (but support) this field and also support a `subnetGroup` field that is a `IClusterSubnetGroup`.

Reply:
Can we actually do it now? A simple `IClusterSubnetGroup` plus a `ClusterSubnetGroup` class with a method `fromSubnetGroupName(): IClusterSubnetGroup` is all I think we need here, and I'd rather do this tiny amount of extra effort now - it's closer aligned to how we model things in the CDK.

Reply:
Sure, fair enough. At the very least, this will allow a user to create a subnet group in the CDK, and then share it among clusters. Worth the extra hour or two; I created the L2.